vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -302,3 +302,42 @@ def test_graph_build_with_description(self, standard_px_chart):
             overlay_style={"visibility": "visible", "opacity": 0.3},
         )
         assert_component_equal(graph, expected_graph, keys_to_strip={"id"})
+
+    def test_graph_build_with_extra(self, standard_px_chart):
+        graph = vm.Graph(
+            figure=standard_px_chart,
+            title="Title",
+            extra={"className": "test"},
```

Comment:
Let's test both: shallow and deep adjustments over the "extra" parameter. 
```suggestion
            extra={"className": "test", "config": {"displayModeBar": False}},
```

Reply:
Good point. 👍